sources/engine/Stride.Games/GameWindow.cs

```diff
@@ -304,6 +304,16 @@ protected void OnFullscreenToggle(object source, EventArgs e)
             IsFullscreen = !IsFullscreen;
         }
 
+        protected void OnDisableFullScreen(object source, EventArgs e)
+        {
+            IsFullscreen = false;
+        }
+
+        protected void EnableFullscreen(object source, EventArgs e)
```

Comment:
Is this method referenced anywhere?

Reply:
No, I can remove it. I left it there just in case someone were to inherit the class but its currently useless.

Reply:
It's signature looks like an event handler. I imagine it's easier for a consumer to run `IsFullscreen = true` instead of invoking this method.

Reply:
It was meant for an event, you are right. The communication between the window and form is very oddly done. Instead of having a reference to the GameWindow directly its awkwardly linked though event.

I removed this one for now since it wasnt a relevent event and users can inherit from the Gamewindow class to add their own if needed.